diff --git a/CMakeLists.txt b/CMakeLists.txt
index cb15e9d1..6f09ba65 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,6 +1,8 @@
+cmake_minimum_required(VERSION 3.3 FATAL_ERROR)
 project(Phonon)
 
-cmake_minimum_required(VERSION 2.8.9 FATAL_ERROR)
+cmake_policy(SET CMP0022 OLD)
+
 add_definitions(-DPHONON_BUILD_WITH_CMAKE)
 
 option(PHONON_ASSERT_STATES "Enable code to assert backend state transitions" ON)

